terraform/publish.tf

```diff
@@ -13,8 +13,8 @@ resource "aws_lambda_function" "async_publish" {
 
   source_code_hash = filebase64sha256(data.archive_file.async_publish_zip.output_path)
 
-  runtime = "provided"
-  layers  = [aws_lambda_layer_version.deno_layer.arn, "arn:aws:lambda:${var.region}:553035198032:layer:git-lambda2:6"]
+  runtime = "provided.al2"
+  layers  = [aws_lambda_layer_version.deno_layer.arn, "arn:aws:lambda:${var.region}:553035198032:layer:git-lambda2:8"]
```

Comment:
so the rhs bump is to an al2 compatible git?
(Hopefully this isn't too arduous for other users, though I guess it is what it is / it needs to be done.)

Reply:
Yup - was using the al2 layer on al1 :upside_down_face: